src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (
+ * see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const arrayStyle = sourceString
+    .split(';')
+    .map(element => element.trim())
+    .filter(el => el.length !== 0);
+
+  arrayStyle.forEach(element => {
+    const key = element.split(':')[0].trim();
+    const value = element.split(':')[1].trim();
```

Comment:
you can create a variable for `element.split (':')` instead of splitting the element twice